firefox-ios/Client/TabManagement/Tab.swift

```diff
@@ -1009,6 +1014,8 @@ class TabWebView: WKWebView, MenuHelperWebViewInterface, ThemeApplicable {
     private weak var delegate: TabWebViewDelegate?
 
     override var inputAccessoryView: UIView? {
+        guard delegate?.tabWebViewShouldShowAccessoryView(self) ?? true else { return nil }
```

Comment:
The equivalent of this `TabWebView` in the `WebEngine` is `DefaultWKEngineWebView`. There's currently no accessory view implemented over there as it was implemented for credit card and we're not there yet with the engine work, but we could add an override to turn it off as you've done here 🤔 And when CC is implemented in the `WebEngine` we can change the check to something else. What do you think?